Transform: adding missing "table"-transform and "series to rows"-transform to Grafana v7-transforms. (#26042)
grafana/grafana: docs/sources/panels/transformations.md

### Merge

> Use this transformation to combine the result from multiple queries into one single result. This is helpful when using the table panel visualization. Values that is mergeable will be combined into the same row. They are considered to be mergeable if the shared fields contains the same data.

Reviewer: ```suggestion
Use this transformation to combine the result from multiple queries into one single result. This is helpful when using the table panel visualization. Values that can be merged are combined into the same row. Values are mergeable if the shared fields contains the same data.
```

{{< docs-imagebox img="/img/docs/transformations/labels-to-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}

> ## Series to rows

Reviewer: ```suggestion
## Series to rows

```

Use this transformation to combine the result from multiple time series data queries into one single result. This is helpful when using the table panel visualization. 

> The result from this transformation will contain three columns: `Time`, `Metric` and `Value`. The `Metric` column is added so you easily can see from what query the metric originate from. You can be customized this value by defining `Label` on the source query.

Reviewer: Are you referring to an example here? Or will this be true in all cases? The way it is written, it seems true in all cases. If that is the case, then you're good.

Author: Yes, it is true for all cases

> The result from this transformation will contain three columns: `Time`, `Metric` and `Value`. The `Metric` column is added so you easily can see from what query the metric originate from. You can be customized this value by defining `Label` on the source query.

Reviewer: ```suggestion
The result from this transformation will contain three columns: `Time`, `Metric`, and `Value`. The `Metric` column is added so you easily can see from which query the metric originates from. Customize this value by defining `Label` on the source query.
```

> In the example below, we are visualizing multiple queries returning time series data before applying the transformation.

Reviewer: ```suggestion
In the example below, we apply a transformation to multiple queries returning time series data.
```

{{< docs-imagebox img="/img/docs/transformations/time-series-before-merge-7-1.png" class="docs-image--no-shadow" max-width= "1100px" >}}

> Here is the same example after applying the merge transformation.

Reviewer: ```suggestion
Here is the same example after applying the Series to rows transformation.
```
